packages/@azure/servicebus/data-plane/test/streamingReceiver.spec.ts

```diff
@@ -777,7 +777,9 @@ describe("Settle an already Settled message throws error", () => {
     await testSettlement(DispositionType.deadletter);
   });
 
-  it("Partitioned Topics and Subscription: deadLetter()", async function(): Promise<void> {
+  it("Partitioned Topics and Subscription: deadLetter() throws error", async function(): Promise<
```

Comment:
deadletter() throws error... ok, but when?

Reply:
The test comes under this block.
> describe("Streaming Receiver - Settle an already Settled message throws error"

When we run the tests, the "Streaming Receiver - Settle an already Settled message throws error" gets printed just before the title of `it(<title>)`. 
It's like 
<pre>  Streaming Receiver - Settle an already Settled message throws error
       Partitioned Topics and Subscription: deadLetter() throws error.  </pre> 

So, this context is enough ?

Reply:
ok, that should be enough